### PR TITLE
nvidia_x11: 495.46 → 510.47.03

### PR DESCRIPTION
**Pull request by fabianhjr**
###### Motivation for this change

Released today as stable: https://www.nvidia.com/download/driverResults.aspx/186156/en-us

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/os-specific/linux/nvidia-x11/default.nix [hunk 1]
```diff
@@ -19,10 +19,10 @@ rec {
   # Policy: use the highest stable version as the default (on our master).
```
Comment (kevincox) [inline pkgs/os-specific/linux/nvidia-x11/default.nix:19]: This package has no maintainers, please consider adding yourself to the maintainer list if you can help out.

**Review by kevincox — APPROVED** [review]

**Comment by michaeladler** [comment]
@fabianhjr Is there any reason why you didn't update the beta driver? Now the beta driver is older than the stable one, which should never be the case. (I just checked because I'm having issues with the beta driver).

**Comment by fabianhjr** [comment]
@michaeladler, afaik there is no more recent beta driver available (at least not the day I checked this PR)

**Comment by fabianhjr** [comment]
Yep, still the case:
image


  | Linux x64 (AMD64/EM64T) Display Driver | 510.47.03 | February 1, 2022
-- | -- | -- | --
  | Linux x64 (AMD64/EM64T) Display Driver | 470.103.01 | January 31, 2022
  | Linux x64 (AMD64/EM64T) Display Driver BETA | 510.39.01 | January 11, 2022

[Linux x64 (AMD64/EM64T) Display Driver](https://www.nvidia.com/download/driverResults.aspx/186156/en-us) NVIDIA Certified 	510.47.03 	February 1, 2022
	[Linux x64 (AMD64/EM64T) Display Driver](https://www.nvidia.com/download/driverResults.aspx/186016/en-us) NVIDIA Certified 	470.103.01 	January 31, 2022
	[Linux x64 (AMD64/EM64T) Display Driver](https://www.nvidia.com/download/driverResults.aspx/184911/en-us) BETA 	510.39.01 	January 11, 2022

**Comment by michaeladler** [comment]
Ah okay, I see. I guess it's correct then, although not really intuitive. Maybe we should add a `latest` alias (similar to `production`) which always selects the latest greatest version - which is the behavior that I wanted when I switched to the beta drivers.

**Pull request merged**